Gems/PhysX/Code/Tests/PhysXColliderComponentModeTests.cpp

```diff
@@ -808,4 +794,134 @@ namespace UnitTest
 
         EXPECT_THAT(newColliderRotation, testing::Not(IsCloseTolerance(capsuleRotation, ManipulatorTolerance)));
     }
+
+    class ColliderPickingFixture : public PhysXEditorColliderComponentManipulatorFixture
+    {
+    public:
+        void SetUpEditorFixtureImpl() override;
+        //! Clicks at a given position and returns the entities that are selected.
+        AzToolsFramework::EntityIdList ClickAndGetSelectedEntities(AzFramework::ScreenPoint screenPoint);
+
+        static inline const float uniformScale = 1.0f;
+        static inline const AZ::Quaternion ShapeRotation = AZ::Quaternion(0, 0, 0, 1);
+        static inline const AZ::Quaternion EntityRotation = AZ::Quaternion(0, 0, 0, 1);
+        static inline const AZ::Vector3 ShapeOffset = AZ::Vector3(0, 0, 0);
+        static inline const AZ::Vector3 EntityTranslation = AZ::Vector3(5.0f, 15.0f, 10.0f);
+    };
```

Comment:
@amzn-ahmadkrm I moved the collider picker tests to PhysXColliderComponentModeTests.cpp for now to make it easier, because it was using the same fixture as the component mode ones. After this change goes in we can look into how to restructure it in a way that it makes sense for both primitive and mesh colliders.

Reply:
That looks good here for now :)